roles/oneview_label/files/oneview_config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ip": "<ip>",
+    "credentials": {
+      "userName": "<userName>",
+      "password": "<password>"
+    },
+    "image_streamer_ip": "",
+    "api_version": 2600,
```

Comment:
since we are validating our resources for API2800, please change this to 2800 and validate against OV 6.10 to avoid duplication of work

Reply:
done